AK/FixedArray.h

```diff
@@ -105,8 +112,14 @@ class FixedArray {
     Span<T> span() { return { data(), size() }; }
 
 private:
-    size_t m_size { 0 };
-    T* m_elements { nullptr };
+    explicit FixedArray(size_t size, T* elements)
+        : m_size(size)
+        , m_elements(elements)
+    {
+    }
+
+    size_t m_size;
+    T* m_elements;
```

Comment:
still need to initialize these for the default ctor to work properly

Reply:
Hah, thanks.